### DISCUSSION TITLE
Profile flagged, 404 error

### DISCUSSION THREAD

**Discussion by akshaypandey89** [community/community#113297] [New to GitHub]
I have access to my account, but it's restricted. Commits are being fetched into repositories but not being deployed, and I can't deploy anything either. Can someone help me? It's been seven days since I opened a request on my primary account, but support isn't responding. I don't know why my account is flagged. Do you have any solutions?

**Comment by akashpawar43** [comment]
I encountered the same issue one month ago, but after filing a complaint, they initially denied my request to unflag my account. However, after some time, my account reappeared in public within four or five days.
you can go here https://support.github.com/contact/account are rise an complaint

**Comment by ettaboyle** [comment]
@akashpawar43 & @akshaypandey89 -- I've seen multiple posts with your accounts asking basic questions and providing answers. This behavior is consistent with Achievements or Badge Hunting. 

**We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges.** You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
